templates/csharp/sso-tab/Components/Welcome.razor

```diff
@@ -55,27 +52,23 @@ else
 @code {
     string userName;
     string errorMessage;
-    bool isInTeams;
     bool isLoading = true;
 
     protected override async Task OnAfterRenderAsync(bool firstRender)
     {
-        await base.OnAfterRenderAsync(firstRender);
-
-        if (firstRender)
+        try
         {
-            isInTeams = await MicrosoftTeams.IsInTeams();
-
-            if (isInTeams)
+            await base.OnAfterRenderAsync(firstRender);
+            if (firstRender)
             {
                 var user = await teamsUserCredential.GetUserInfoAsync();
                 userName = user.DisplayName;
             }
-            else
-            {
-                errorMessage = "Not running in Microsoft Teams.";
-            }
-
+        }
+        catch (Exception e) { 
+            errorMessage = e.Message;
```

Comment:
What could be some of these error messages? I'm not sure if we've confirmed with PM whether directly displaying these error messages to users is appropriate.

Reply:
Thanks for the heads up, I see that none of the other files deal with the error, I'll correct it.